Jenkinsfile

```
@@ -380,6 +381,94 @@ pipeline {
                         }
                     }
                 }
+                stage('rootless') {
+                    when {
+                        beforeAgent true
+                        expression { params.amd64 && params.rootless }
```

Tempted to say: decouple this option from `params.amd64`

```suggestion
                        expression { params.rootless }
```

---

fixed